thespian/system/transport/errmgmt.py

```diff
@@ -14,7 +14,7 @@ def err_conn_refused(errex):
 
 
 def err_send_inprogress(err):
-    return err in [errno.EINPROGRESS, errno.EAGAIN]
+    return err in [errno.EINPROGRESS, errno.EAGAIN, errno.ENOTCONN]
```

Comment:
Required to pass `thespian/test/test_generators.py` test on MacOS. `selectors` signal socket as ready to send yet `[Errno 57] Socket is not connected` error is raised when sending in `_next_XMIT_2()`. After a slight backoff, all works fine. From what I could tell looking at packet dumps, `socket.send()` was throwing this error _after_ TCP connection was already established. What is special about this test is the use of global names combined with creation of multiple actors in quick succession.

Fun fact: This test was created after report from Rally lead developer in March 2017. Rally no longer uses global names. They were removed from Rally by Thespian lead developer in December 2017. :-)